INFO     root:test_20_sysemailconfig.py:37 ==========开始执行用例：sysemailconfig_verify_01==========
INFO     root:runcase.py:30 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 START-·-·-·-·-·-·-·-·-·-
INFO     root:handleyaml.py:94 Yaml测试用例基础架构检查通过
INFO     root:runcase.py:47 重组后的用例信息：{'name': '系统邮箱验证', 'description': '系统邮箱验证', 'request': {'data': {'mail.host': '192.168.154.30', 'mail.password': '123456', 'mail.port': '25', 'mail.securityLink': 'false', 'mail.username': 'gj1@gaojie.com'}, 'method': 'POST', 'address': '/api/auth/v1/system/verifyMailData', 'headers': {'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6ImY2MzRmNDkyLTM0NjktNGU2Yi05NTI4LTQwNGQxOTdjZjBkNyIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiJmNjM0ZjQ5Mi0zNDY5LTRlNmItOTUyOC00MDRkMTk3Y2YwZDciLCJleHAiOjE2ODc5NDAzNDIsImlhdCI6MTY4Nzc2MDM0Mn0.AlXTVLxZrmNgDRvtdV-Lwy0OeP8VGt-QOwjVWR7TCo1cHLTNKZ39YvetKwmd8KQDqe-AscMfoPux5PiXN1rafQ', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/systemEmailConfig'}}, 'validate': {'code': 200, 'response': {'result': 0}}, 'epic': '系统管理', 'feature': '参数管理-系统邮箱配置', 'story': '系统邮箱验证接口', 'base_url': 'https://192.168.148.174:31000', 'caseid': 'sysemailconfig_verify_01'}
INFO     root:runcase.py:83 请求地址：https://192.168.148.174:31000/api/auth/v1/system/verifyMailData
INFO     root:runcase.py:84 请求方法：POST
INFO     root:runcase.py:85 请求头：{'Content-Type': 'application/json', 'token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJhdWQiOiJhcHBsaWNhdGlvbiIsInN1YiI6ImY2MzRmNDkyLTM0NjktNGU2Yi05NTI4LTQwNGQxOTdjZjBkNyIsImlzcyI6ImJtc19zZXJ2ZXIiLCJzZXNzaW9uSWQiOiJmNjM0ZjQ5Mi0zNDY5LTRlNmItOTUyOC00MDRkMTk3Y2YwZDciLCJleHAiOjE2ODc5NDAzNDIsImlhdCI6MTY4Nzc2MDM0Mn0.AlXTVLxZrmNgDRvtdV-Lwy0OeP8VGt-QOwjVWR7TCo1cHLTNKZ39YvetKwmd8KQDqe-AscMfoPux5PiXN1rafQ', 'Referer': 'https://192.168.148.174:31000/sub-app-unity/systemEmailConfig'}
INFO     root:runcase.py:95 请求参数：{'mail.host': '192.168.154.30', 'mail.password': '123456', 'mail.port': '25', 'mail.securityLink': 'false', 'mail.username': 'gj1@gaojie.com'}
DEBUG    urllib3.connectionpool:connectionpool.py:1003 Starting new HTTPS connection (1): 192.168.148.174:31000
DEBUG    urllib3.connectionpool:connectionpool.py:456 https://192.168.148.174:31000 "POST /api/auth/v1/system/verifyMailData HTTP/1.1" 200 52
INFO     root:runcase.py:157 请求接口结果： {'result': 0, 'message': '验证成功，参数可用'}
INFO     root:checkresult.py:113 code断言通过, 期望状态码:200, 实际状态码:200
INFO     root:checkresult.py:48 返回结果断言通过, 期望返回值:{'result': 0}, 实际返回值:{'result': 0, 'message': '验证成功，参数可用'}
INFO     root:runcase.py:63 -·-·-·-·-·-·-·-·-·-发送请求并接受信息 END-·-·-·-·-·-·-·-·-·-